Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m

```diff
@@ -106,7 +106,7 @@ - (void)setEnableFocusRing:(BOOL)enableFocusRing {
 - (void)scrollViewDidScroll:(RCTUIScrollView *)scrollView // TODO(macOS ISS#3536887)
 {
   RCTDirectEventBlock onScroll = self.onScroll;
-
+  [_scrollView setHasVerticalScroller:YES];
```

Comment:
Do we want this to be configurable at a JS level? I don't necessarily have a strong preference either way, but it could be an interesting "nice-to-have."